src/app/i-have-corona/checklist/checklist.component.ts

```diff
@@ -13,20 +13,74 @@ export class ChecklistComponent implements OnInit {
     ['Diarree', 1],['Blindvliesontsteking', 1], ['Kortademigheid of ademhalingsproblemen', 1],
     ['Pijn of druk op de borst', 1], ['Aangetast spraak- of bewegingsvermogen', 1]];
 
-  startPage: Boolean = true;
+  startPage : Boolean = true;
   coronaConfirmedPage : Boolean = false;
   symptomsPage : Boolean = false;
+  passwordPage : Boolean = false;
+  noSymptomsPage : Boolean = false;
+  lowSymptomConfirmationPage : Boolean = false;
+  wrongPassword : Boolean = false;
```

Comment:
Do you need all of these booleans.
For choices between 2 please use: (True: page1 ; False: page2)
If that is not possible maybe use an array?